### PR TITLE
Created a test folder Find and rootFolderTest which contains files wh…

### PR DESCRIPTION
**Pull request by utsav-sharma2020**
…ich may/maynot be ignored. The MainAttempt.java is the program which runs the feature of gitignore

Please ensure your pull request adheres to the following guidelines:

- **Title**: A brief, descriptive summary of your changes.
- **Description**: Explain the changes you've made and why you've made them.
- **Branch Naming**: Prefix the type of change in the branch name and/or select the option relevant:
  - [ ] `chore/` This change requires a documentation update
  - [ ] Bug fix `bugfix/` (non-breaking change which fixes an issue)
  - [x] New feature `feature/` (non-breaking change which adds functionality)
  - [ ] Breaking change `fix/` (fix or feature that would cause existing functionality to not work as expected)
- **Related Issue(s)**: Mention any issues that are related to these changes.
- **Testing**: Detail the testing that was done to validate the changes.
- **Checklist**:
  - [x] I have performed a self-review of my own code.
  - [ ] I have added tests that prove my fix is effective or that my feature works.
- **Screenshots/Output**: Include any relevant screenshots or output demonstrating the changes (if applicable).
- **Additional Notes**: Any further comments or notes for the reviewers.

### CODE REVIEW

**Review by tanmayr71 — CHANGES_REQUESTED** [review]
Code looks good, can you remove the .DS_Store files annd create a draft again? Thanks!

**Review by Hao-191 — COMMENTED** [review]
looks good

File: src/main/java/Find/.DS_Store
Comment (sash7410) [inline src/main/java/Find/.DS_Store:1]: Can we not push .DS_Store files?

File: src/main/java/.DS_Store
Comment (sash7410) [inline src/main/java/.DS_Store:1]: Can we not push .DS_Store files?

**Comment by sash7410** [comment]
LGTM , just delete .DSstore files

**Comment by nt591** [comment]
Likely you'll need to update the .gitignore file in the repo to avoid this from uploading in the future.

**Review by slachitoff — COMMENTED** [review]
I agree with the previous comments. Apart from adding the .DS_Store files to the .gitignore, the code itself looks good for now.

**Comment by sash7410** [comment]
I have added the .DS_Store in the gitignore file in the new PR I made, can we merge that branch?

**Comment by tanmayr71** [comment]
Should we close this PR? This work has been done by @Hao-191 
@utsav-sharma2020 - Can you check the work the work done already and see if we need this commit?

**Pull request closed without merging**